crates/types/src/storage.rs

```diff
@@ -231,6 +228,44 @@ impl StorageEncode for String {
         Ok(())
     }
 }
+impl StorageDecode for String {
+    fn decode<B: ::bytes::Buf>(
```

Comment:
I noticed that `bytes` is already imported, was this intentionally being explicit for readability?